Author: Cesare Falco <c.falco@ubuntu.com>
Description: Man page revision.
 Updated man pages with new switches.
Forwarded: no

Index: mame/src/osd/sdl/man/mame.6
===================================================================
--- mame.orig/src/osd/sdl/man/mame.6	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/mame.6	2013-09-29 18:44:27.000000000 +0200
@@ -13,7 +13,7 @@
 .\" and updated by Andrew Burton <burtona@gol.com>, July 2003
 .\"
 .\"
-.TH MAME 6 2013-06-12 0.149 "MAME \- The Multiple Arcade Machine Emulator"
+.TH MAME 6 2013-09-23 0.150 "MAME \- The Multiple Arcade Machine Emulator"
 .\"
 .\"
 .\" NAME chapter
@@ -1051,6 +1051,12 @@
 .SS Misc options
 .\" *******************************************************
 .TP
+.B \-drc
+Enable DRC cpu core if available. Enabled by default.
+.TP
+.B \-drc_use_c
+Force DRC use C backend.
+.TP
 .B \-bios \fIbiosname
 Specifies the specific BIOS to use with the current game, for game
 systems that make use of a BIOS. The \-listxml output will list all of
@@ -1093,6 +1099,15 @@
 .TP
 .B \-autoboot_script, \-script \fR[\fIfilename.lua\fP]
 File containing scripting to execute after machine boot.
+.TP
+.B \-http
+Enable local http server. Disabled by default.
+.TP
+.B \-http_port \fIport
+Internal webserver listener port.
+.TP
+.B \-http_path \fIpath
+Path to web files. Default is /usr/share/games/mess/web.
 .\" +++++++++++++++++++++++++++++++++++++++++++++++++++++++
 .\" SDL specific
 .\" +++++++++++++++++++++++++++++++++++++++++++++++++++++++
@@ -1102,6 +1117,11 @@
 if it detects that the emulation has locked up.
 .\"
 .\"
+.\" SEE ALSO chapter
+.SH SEE ALSO
+chdman(1), jedutil(1), ldresample(1), ldverify(1), romcmp(1), testkeys(1)
+.\"
+.\"
 .\" LEGAL NOTICE chapter
 .SH LEGAL NOTICE
 Please visit the MAME website for some important legal information:
Index: mame/src/osd/sdl/man/chdman.1
===================================================================
--- mame.orig/src/osd/sdl/man/chdman.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/chdman.1	2013-09-29 18:38:30.000000000 +0200
@@ -6,7 +6,7 @@
 .\" Ashley T. Howes <debiandev@ashleyhowes.com>, February 2005
 .\" updated by Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH CHDMAN 1 2012-12-01 0.147u3 "MAME Compressed Hunks of Data (CHD) manager"
+.TH CHDMAN 1 2013-09-23 0.150 "MAME Compressed Hunks of Data (CHD) manager"
 .\"
 .\" NAME chapter
 .SH NAME
@@ -15,7 +15,7 @@
 .\" SYNOPSIS chapter
 .SH SYNOPSIS
 .B chdman
-.I option switches
+.I option
 .\"
 .\" DESCRIPTION chapter
 .SH DESCRIPTION
@@ -251,3 +251,5 @@
 .TP
 .B \-\-verbose, \-v
 Output additional information about the operation.
+.SH SEE ALSO
+mame(6), mess(6)
Index: mame/src/osd/sdl/man/imgtool.1
===================================================================
--- mame.orig/src/osd/sdl/man/imgtool.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/imgtool.1	2013-09-29 18:39:13.000000000 +0200
@@ -6,12 +6,12 @@
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2011
 .\"
 .\"
-.TH imgtool 1 2012-12-01 0.147u3 "Image manipulation tool for MESS"
+.TH IMGTOOL 1 2013-09-23 0.150 "MESS media image manipulation tool"
 .\"
 .\"
 .\" NAME chapter
 .SH NAME
-imgtool \- A generic image manipulation tool for MESS
+imgtool \- MESS media image manipulation tool
 .\"
 .\"
 .\" SYNOPSIS chapter
@@ -154,4 +154,4 @@
 .\"
 .\" SEE ALSO chapter
 .SH SEE ALSO
-.BR mess (6)
+mess (6)
Index: mame/src/osd/sdl/man/mess.6
===================================================================
--- mame.orig/src/osd/sdl/man/mess.6	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/mess.6	2013-09-29 18:53:11.000000000 +0200
@@ -16,7 +16,7 @@
 .\" http://www.mess.org/
 .\"
 .\"
-.TH MESS 6 2013-06-12 0.149 "The Multiple Emulator Super System (MESS)"
+.TH MESS 6 2013-09-23 0.150 "The Multiple Emulator Super System (MESS)"
 .\"
 .\"
 .\" NAME chapter
@@ -1033,6 +1033,12 @@
 .SS Misc options
 .\" *******************************************************
 .TP
+.B \-drc
+Enable DRC cpu core if available. Enabled by default.
+.TP
+.B \-drc_use_c
+Force DRC use C backend.
+.TP
 .B \-bios \fIbiosname
 Specifies the specific BIOS to use with the current system, for
 systems that make use of a BIOS. The \-listxml output will list all of
@@ -1076,6 +1082,15 @@
 .B \-autoboot_script, \-script \fR[\fIfilename.lua\fP]
 File containing scripting to execute after machine boot.
 .TP
+.B \-http
+Enable local http server. Disabled by default.
+.TP
+.B \-http_port \fIport
+Internal webserver listener port.
+.TP
+.B \-http_path \fIpath
+Path to web files. Default is /usr/share/games/mess/web.
+.TP
 .B \-newui, \-nu
 Use the new MESS UI.
 .TP
@@ -1084,6 +1099,11 @@
 if it detects that the emulation has locked up.
 .\"
 .\"
+.\" SEE ALSO chapter
+.SH SEE ALSO
+castool(1), chdman(1), imgtool(1), testkeys(1)
+.\"
+.\"
 .\" LEGAL NOTICE chapter
 .SH LEGAL NOTICE
 Please visit the MAME website for some important legal information:
Index: mame/src/osd/sdl/man/jedutil.1
===================================================================
--- mame.orig/src/osd/sdl/man/jedutil.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/jedutil.1	2013-09-29 18:39:54.000000000 +0200
@@ -8,21 +8,16 @@
 .\" References
 .\" http://aarongiles.com/?p=159
 .\"
-.TH JEDUTIL 1 2012-12-01 0.147u3 "MAME JED files manager"
+.TH JEDUTIL 1 2013-09-23 0.150 "MAME JEDEC file utilities"
 .\"
 .\" NAME chapter
 .SH NAME
-jedutil \- convert a .JED file into its binary form, and vice\-versa
+jedutil \- MAME JEDEC file utilities
 .\"
 .\" SYNOPSIS chapter
 .SH SYNOPSIS
 .B jedutil
-.IR source ".jed " target .bin
-.br
-.B jedutil
-.IR source ".bin " target .jed
-.br
-.B jedutil help
+.RI [ option ]
 .\"
 .\" DESCRIPTION chapter
 .SH DESCRIPTION
@@ -36,3 +31,20 @@
 .PP
 .B jedutil
 provides an easy way to convert PLDs data between these two formats.
+.\"
+.\" OPTIONS chapter
+.SH OPTIONS
+.TP
+.B \-convert \fIinputfile outputfile \fR[\fIfuses\fR]
+Converts JEDEC data between file formats. Both .jed (JEDEC) and .pla
+(Berkeley standard PLA) files can be converted to a .bin (binary) file,
+while a binary file can be converted to a .jed file only.
+.TP
+.B \-view \fIinputfile device
+Dump logic equations. \fIinputfile\fR can be either a JED or binary file.
+.TP
+.B \-viewlist
+View list of supported devices.
+.SH SEE ALSO
+mame(6), mess(6)
+
Index: mame/src/osd/sdl/man/ldresample.1
===================================================================
--- mame.orig/src/osd/sdl/man/ldresample.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/ldresample.1	2013-09-29 18:41:29.000000000 +0200
@@ -3,7 +3,7 @@
 .\" Second parameter, SECTION, should be 1-8, maybe w/ subsection
 .\" other parameters are allowed: see man(7), man(1)
 .\"
-.TH ldresample 1 2012-12-01 0.147u3 "MESS Laserdisc audio manipulation tool"
+.TH LDRESAMPLE 1 2013-09-23 0.150 "MAME laserdisc audio manipulation tool"
 .\"
 .\" Please adjust this date whenever revising the manpage.
 .\"
@@ -19,7 +19,7 @@
 .\" for manpage-specific macros, see man(7)
 
 .SH "NAME"
-ldresample \- Laserdisc audio synchronizer and resampler.
+ldresample \- MAME laserdisc audio manipulation tool
 
 .SH "SYNOPSIS"
 .B ldresample
@@ -32,26 +32,18 @@
 .B [slope]
 
 .SH "DESCRIPTION"
-Assists in resynchronizing audio tracks in a CHD (Compressed Hunks of Data) file
- with frames.
-.br
+Assists in resynchronizing audio tracks in a CHD (Compressed Hunks of Data)
+file with frames.
+.PP
 .B Offset
-parameter and 
-.B slope
-option make a linear equation f(x) which
-describes the sample offset from the source as a function
-of field number.
-
+parameter and \fBslope\fR option make a linear equation f(x) which
+describes the sample offset from the source as a function of field number.
 .SH "COPYRIGHT"
 Copyright \(co 2008 Aaron Giles. BSD License.
-
-.SH "AUTHOR"
-Written by Aaron Giles.
-
 .SH "SEE ALSO"
-.BR ldverify (1),
-.BR ldplayer (1).
-
-.sp 3
+ldverify(1), mame(6), mess(6)
+.SH "AUTHORS"
+Written by Aaron Giles.
+.PP
 This manual page was written by Ludovic Lechapt <ludomatic@gmail.com>,
 for the Debian project but may be used by others.
Index: mame/src/osd/sdl/man/ldverify.1
===================================================================
--- mame.orig/src/osd/sdl/man/ldverify.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/ldverify.1	2013-09-29 18:42:00.000000000 +0200
@@ -5,7 +5,7 @@
 .\" Man page created from source and usage information by 
 .\" Cesare Falco <c.falco@ubuntu.com>, August 2008
 .\"
-.TH LDVERIFY 1 2012-12-01 0.147u3 "MAME laserdisc data checker"
+.TH LDVERIFY 1 2013-09-23 0.150 "MAME laserdisc data checker"
 .\"
 .\" NAME chapter
 .SH NAME
@@ -21,4 +21,5 @@
 is a tool which walks through either an AVI or CHD capture from a
 laserdisc and ensures that there are no anomalies lurking in
 the VBI data.
-
+.SH SEE ALSO
+ldresample(1), mame(6), mess(6)
Index: mame/src/osd/sdl/man/romcmp.1
===================================================================
--- mame.orig/src/osd/sdl/man/romcmp.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/romcmp.1	2013-09-29 18:50:27.000000000 +0200
@@ -9,7 +9,7 @@
 .\" References
 .\" http://www.mame.net/mamefaq.html
 .\"
-.TH ROMCMP 1 2012-12-01 0.147u3 "MAME romset checking tool"
+.TH ROMCMP 1 2013-09-23 0.150 "MAME romset checking tool"
 .\"
 .\" NAME chapter
 .SH NAME
@@ -24,15 +24,14 @@
 .\"
 .\" DESCRIPTION chapter
 .SH DESCRIPTION
-An eternal nuisance in arcade emulation are incomplete or corrupt dumps of the ROM chips.
-\fBromcmp\fP is a tool developed to detect the most common errors that can occur when dumping ROMS,
-for example stuck bits and address line errors.
+\fBromcmp\fP is a tool developed to detect the most common errors that
+can occur when dumping ROMS, for example stuck bits and address line errors.
 .PP
-It can check the ROMs in ZIP files or subdirectories.
-.PP
-\fBromcmp\fP can also be used to compare two ROM sets, by giving it two files or subdirectories
-as parameters. It will determine which ROMS are identical and which are the closest matches.
-This is useful when figuring out if a newly dumped ROM set is a clone of another ROM set.
+It can check the ROMs in ZIP files or subdirectories; it can also be used
+to compare two ROM sets, by giving it two files or subdirectories
+as parameters. It will determine which ROMS are identical and which are
+the closest matches.  This is useful when figuring out if a newly dumped
+ROM set is a clone of another ROM set.
 .\"
 .\" OPTIONS chapter
 .SH OPTIONS
@@ -42,11 +41,16 @@
 .\"
 .\" EXAMPLES chapter
 .SH EXAMPLES
-.B romcmp \fIjumpkids.zip\fP
+.B romcmp jumpkids.zip
 will output the following:
 
   11 files
   23.3c			FIXED BITS (xxxxxx1x)
   23.3c		  	FIRST AND SECOND HALF IDENTICAL
       
-This tells us that the 23.3C ROM in Jump Kids is bad, and incidentally, it causes the missing sound in that game.
+This tells us that the 23.3C ROM in Jump Kids is bad, and incidentally,
+it causes the missing sound in that game.
+.\"
+.\" SEE ALSO chapter
+.SH SEE ALSO
+mame(6)
Index: mame/src/osd/sdl/man/testkeys.1
===================================================================
--- mame.orig/src/osd/sdl/man/testkeys.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/testkeys.1	2013-09-29 18:51:52.000000000 +0200
@@ -5,11 +5,11 @@
 .\" Man page created from source and usage information
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2007
 .\"
-.TH TESTKEYS 1 2012-12-01 0.147u3 "MAME SDL keycode checker"
+.TH TESTKEYS 1 2013-09-23 0.150 "MAME SDL keycode scanner"
 .\"
 .\" NAME chapter
 .SH NAME
-testkeys \- returns SDL keycode for key pressed.
+testkeys \- MAME SDL keycode scanner
 .\"
 .\" SYNOPSIS chapter
 .SH SYNOPSIS
@@ -17,7 +17,11 @@
 .\"
 .\" DESCRIPTION chapter
 .SH DESCRIPTION
-Since 0.113 SDLMAME introduced keymap files to handle keyboards with non\-us layouts.
+Since 0.113 (SDL)MAME introduced keymap files to handle keyboards
+with non\-us layouts.
 .PP
-This simple utility helps determining which SDL keycode is bound to each key, thus
-making the process of compiling keymap files a bit easier.
+This simple utility helps determining which SDL keycode is bound
+to each key, thus making the process of compiling keymap files a
+bit easier.
+.SH SEE ALSO
+mame(6), mess(6)
Index: mame/src/osd/sdl/man/castool.1
===================================================================
--- mame.orig/src/osd/sdl/man/castool.1	2013-09-29 18:26:53.000000000 +0200
+++ mame/src/osd/sdl/man/castool.1	2013-09-29 18:37:52.000000000 +0200
@@ -6,7 +6,7 @@
 .\" Cesare Falco <c.falco@ubuntu.com>, February 2011
 .\"
 .\"
-.TH castool 1 2012-12-01 0.147u3 "MESS Generic cassette manipulation tool"
+.TH CASTOOL 1 2013-09-23 0.150 "MESS Generic cassette manipulation tool"
 .\"
 .\"
 .\" NAME chapter
@@ -33,7 +33,7 @@
 .B convert
 Convert a cassette dump to a .wav audio file. See
 .B castool 
-(with no arguments) for a full list of supported cassette formats.
+(without arguments) for a full list of supported cassette formats.
 .\"
 .\"
 .\" EXAMPLES chapter
@@ -43,4 +43,4 @@
 .\"
 .\" SEE ALSO chapter
 .SH SEE ALSO
-.BR mess (6)
+mess(6)
